readme.markdown

```diff
@@ -1103,6 +1103,7 @@ Nimble includes a ``publish`` command which does this for you automatically.
   listed in ``bin``. Possible values include: ``c``, ``cc``, ``cpp``, ``objc``,
   ``js``.
   **Default**: c
+* ``paths`` - A list of relative paths that will be expanded on `nimble.paths` and the search paths options to the compiler.
```

Comment:
these would always be relative to the nimble file?

Reply:
can they be used to escape the package sandbox? ie if I do `../../../etc`, can I get access to `/etc/passwd` with `staticRead` in a malicious package?

Reply:
Doesnt `staticRead` uses the `appDir` as root? 

This PR would just add another directory to the compiler search path, which you could previously access anyways.
For instance, previously the directory would be something like: 

`/Volumes/Store/Projects/testclient/nimbledeps/pkgs2/nim-2.1.1-a4e01d628f14da66e92b8d2d41d7c0ec48c3bb05`

If you have `paths = @["compiler"]` it will get expanded to: 

```
--path:"/Volumes/Store/Projects/testclient/nimbledeps/pkgs2/nim-2.1.1-a4e01d628f14da66e92b8d2d41d7c0ec48c3bb05"
--path:"/Volumes/Store/Projects/testclient/nimbledeps/pkgs2/nim-2.1.1-a4e01d628f14da66e92b8d2d41d7c0ec48c3bb05/compiler"
```

Reply:
aka what would `paths = @["../../compiler"]` expand  to?